docs/includes/prerequisites.rst

```diff
@@ -0,0 +1,47 @@
+.. Prerequisites for using JDBC connector
+
+**Prerequisites:**
+
+- :ref:`Confluent Platform <installation>` is installed and running by using the Confluent CLI. This quick start assumes that you are using the Confluent CLI, but standalone installations are also supported. By default ZooKeeper, Kafka, Schema Registry, Kafka Connect REST API, and Kafka Connect are started with the ``confluent start`` command. For more information, see :ref:`installation_archive`.
```

Comment:
How about: 
_Confluent Platform is installed. **Services are** running by using the Confluent CLI_?

to be extra clear that CP is **not** installed by CLI?
(not sure about the full stop between the two sentences - your call in general)